web/docker/Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN wget --progress=:giga https://github.com/WebAssembly/binaryen/releases/downl
     mv wasm-opt /usr/local/bin
 
 # Installing Rust using rustup:
-RUN wget 'https://sh.rustup.rs' --quiet -O- | sh -s -- -y --profile minimal --target wasm32-unknown-unknown
+RUN wget 'https://sh.rustup.rs' --quiet -O- | sh -s -- -y --profile minimal --target wasm32-unknown-unknown && rustup component add rust-src
```

Comment:
I think the installer script for rustup can take the `--component` flag immediately, no need to invoke it separately.